lib/screen/weather_screen.dart

```diff
@@ -16,8 +16,43 @@ class _WeatherScreenState extends State<WeatherScreen> {
   String _weatherImage = '';
 
   void _reloadWeather() {
-    _weatherImage = _weather.fetchSimpleWeather();
-    setState(() {});
+    try {
+      _weatherImage = _weather.fetchThrowsWeather('tokyo');
+      setState(() {});
+    } on YumemiWeatherError catch (error) {
+      _convertYumemiWeatherError(error);
```

Comment:
[nits]
命名的に _convertYumemiWeatherError は変換処理をするだけのメソッドにみえるので、戻り値を String にして次のようにするとよりわかりやすくなりそうな気がします！
```suggestion
      final errorMessage = _convertYumemiWeatherError(error);
      _openErrorDialog(errorMessage);
```

Reply:
確かにその書き方のほうがわかりやすいですね！とても勉強になります！！